requirements/app.in

```diff
@@ -50,7 +50,7 @@ marshmallow>=3
 marshmallow-polyfield
 marshmallow-sqlalchemy>=0.23.1
 webargs
-uniplot
+uniplot>=0.7.0
```

Comment:
I couldn't see how you use uniplot differently now. It would probably still work with <0.7.0, just not if NaN is in the data. So, do we technically need to depend on a higher version, or do we like this feature to be around? Always good to add the reasoning as a comment to make this clear for later.

Reply:
I would like this feature to be around. When selecting multiple sensors for some time period, having NaN data in that set is very common. With previous uniplot versions, the time series will not align right, which is a serious problem to me.

From uniplot's `setup.py` (version 0.7.0):
```
python_requires=">=3.5",
install_requires=["numpy>=1.15.0"],
```
which are older versions than the first ones we started using for this repo as far back as I can tell (https://github.com/FlexMeasures/flexmeasures/commit/10cff9fd4a34976405b105071fd6fa5febde60c4).

I'll add an inline note to where we define the minimum version requirement.